File: solutions/989.Add_to_Array-Form_of_Integer/989.Add_to_Array-Form_of_Integer.ts
function addToArrayForm(num: number[], k: number): number[] {
    const sumNumber = (BigInt(num.join('')) + BigInt(k)).toString()
    let arr:number[] = []

    for(let i = 0; i < sumNumber.length; i++) {
        arr.push(Number(sumNumber[i]))
    }
    return arr
};

File: solutions/easy/989.Add_to_Array-Form_of_Integer/989.Add_to_Array-Form_of_Integer.py
class Solution:
    def addToArrayForm(self, num: List[int], k: int) -> List[int]:
        sum_nums = str(int(''.join(map(str, num))) + k)
        arr = []

        for sum_num in sum_nums:
            arr.append(int(sum_num))
        return arr
